velox/exec/Window.h

```diff
@@ -22,6 +22,16 @@
 
 namespace facebook::velox::exec {
 
+// Structure for k preceding/following frame arguments, index stores the index
```

Comment:
Nit -- Rephrase 
"Used for k preceding/following frames. Index is the column index if k is a column. value is used to read the column values from the input, or contains the constant value if k is a constant."